pkg/monitors/cloudfoundry/client.go

```diff
@@ -0,0 +1,114 @@
+package cloudfoundry
+
+import (
+	"context"
+	"crypto/tls"
+	"errors"
+	"log"
+	"net"
+	"net/http"
+	"strings"
+	"time"
+
+	"code.cloudfoundry.org/go-loggregator/rpc/loggregator_v2"
+	"github.com/cloudfoundry/go-loggregator"
+	"github.com/signalfx/golib/v3/datapoint"
+	"github.com/signalfx/signalfx-agent/pkg/core/common/dpmeta"
+	"github.com/signalfx/signalfx-agent/pkg/utils"
+	"github.com/sirupsen/logrus"
+)
+
+const defaultShardID = "signalfx-nozzle"
```

Comment:
Looks like this default does not match the default config, `signalfx_nozzle`. Is that intentional?

Reply:
Do we actually need it, seems like it's always overwritten by the value from the config

Reply:
It's good to have to decouple the client from the code that uses it (e.g. in case that other code gets refactored and that assumption is no longer true that it is always overwritten).